desc/backend.py

```diff
@@ -134,8 +147,62 @@ def wrapper(*args, **kwargs):
 
         return wrapper
 
-    # JAX implementation is not differentiable on gpu.
-    eigh_tridiagonal = execute_on_cpu(jax.scipy.linalg.eigh_tridiagonal)
+    _eigh_tridiagonal = jnp.vectorize(
+        jax.scipy.linalg.eigh_tridiagonal,
+        signature="(m),(n)->(m)",
+        excluded={"eigvals_only", "select", "select_range", "tol"},
+    )
+    if desc_config["kind"] == "gpu":
```

Comment:
using `jax.lax.platform_dependent` might be a better way to go here, especially once #1495 is working

Reply:
(I can't make a wrapper for a function that does not support keyword arguments, when the original function requires keyword args).